.github/workflows/php.yml

```diff
@@ -62,6 +62,11 @@ jobs:
       - name: Install dependencies
         run: composer install --prefer-dist
 
+      - name: Install PHPUnit 8
+        run: curl -O -L https://phar.phpunit.de/phpunit-8.phar
+          && chmod +x phpunit-8.phar
+          && mv phpunit-8.phar /usr/local/bin/phpunit
```

Comment:
What about running tests locally? Do we also need to install PHPUnit separately?

Reply:
There's no difference to the current situation.

But I think you're right that because of "johnkary/phpunit-speedtrap", we pin phpunit at 5.8 in `composer.lock`
Shall I remove this dependency, so the phpunit pinning is deleted as well?

I think the speedtrap isn't necessary anymore because the speed of just the phpunit tests is <1m for PHP8 and ~2-3m for PHP7

Reply:
Ah, and to answer your question: Yes we do need to install phpunit separately, but for me it feels more transparent than force-upgrading a current phpunit version per `--ignore-platform-req`

Reply:
Yes please, and remove the testing dependencies from the composer.json

Reply:
I created a setup-script instead of manually executing the command and added it to a new `README.md` inside the `tests/` directory.